src/main/java/org/geysermc/discordbot/tags/TagsManager.java

```diff
@@ -58,13 +66,16 @@ private static void loadTags() {
                         continue;
                     }
 
+                    tagIterator:
```

Comment:
Could we change this to just use continue and a bool?

Reply:
Just ended up removing the label as it wasn't necessary